cmplibrary/src/test/java/com/sourcepoint/cmplibrary/data/network/model/optimized/MetaDataRespTest.kt

```diff
@@ -1,18 +0,0 @@
-package com.sourcepoint.cmplibrary.data.network.model.optimized
-
-import com.sourcepoint.cmplibrary.data.network.converter.JsonConverter
-import com.sourcepoint.cmplibrary.data.network.converter.converter
-import com.sourcepoint.cmplibrary.util.file2String
-import kotlinx.serialization.decodeFromString
-import kotlinx.serialization.json.encodeToJsonElement
-import org.junit.Test
-
-class MetaDataRespTest {
-
-    @Test
-    fun `GIVEN a MetaData RETURN the parsed obj`() {
```

Comment:
A similar test exists in the core module now, right?

Reply:
Yes, and that's exactly the goal. Network requests/responses will be tested in the core module so they don't have to be re-tested in the native SDKs 🔥